native/packages/react-native-bpk-component-progress/src/BpkProgress.js

```diff
@@ -0,0 +1,160 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* @flow */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  StyleSheet,
+  View,
+  Animated,
+  Easing,
+  Platform,
+  ViewPropTypes,
+} from 'react-native';
+import {
+  animmationDurationBase,
+  spacingMd,
+  colorGray100,
+  colorGreen500,
+  colorGreen50,
+  borderRadiusPill,
+  spacingXxl,
+} from 'bpk-tokens/tokens/base.react.native';
+
+const styles = StyleSheet.create({
+  container: {
+    height: spacingMd,
+    overflow: 'hidden',
```

Comment:
good to know (or not) I guess is worth leaving it here anyways or you think I should just remove it? The fill and track height are the same so I don't think it will make any difference

Reply:
I'd remove it in that case